ambry-cloud/src/test/java/com.github.ambry.cloud/azure/AzureCloudDestinationTest.java

```diff
@@ -413,21 +392,8 @@ public void testGetDeadBlobs() throws Exception {
     assertEquals(1, azureMetrics.deadBlobsQueryTime.getCount());
   }
 
-  /** Test findEntriesSince. */
-  @Test
-  public void testFindEntriesSince() throws Exception {
-    switch (azureReplicationFeedType) {
-      case COSMOS_CHANGE_FEED:
-        testFindEntriesSinceUsingChangeFeed();
-        break;
-      case COSMOS_UPDATE_TIME:
-        testFindEntriesSinceUsingUpdateTime();
-        break;
-    }
-  }
-
   /** Test findEntriesSince when cloud destination uses change feed based token. */
-  private void testFindEntriesSinceUsingChangeFeed() throws Exception {
+  public void testFindEntriesSinceUsingChangeFeed() throws Exception {
```

Comment:
Need Test annotation in these two methods?

Reply:
fixed.